Resources/Prototypes/DeltaV/Entities/Mobs/Species/vulpkanin.yml

```diff
@@ -105,6 +106,13 @@
     understands:
     - GalacticCommon
     - Canilunzt
+  - type: ConsumeDelayModifier
+    foodDelayMultiplier: 0.5
+    drinkDelayMultiplier: 0.5
```

Comment:
@DEATHB4DEFEAT There should probably be a way to 'add a trait' to an Entity outside of the character editor- a list in a Component or something and it just has the same effect as giving it to them. Avoids duplicating values when the intention is clear- this is meant to be the trait.